Dockerfile

```
@@ -39,10 +42,19 @@ WORKDIR $WORKDIR
 RUN apk --no-cache add \
     mediainfo
 
+# Add user to run the application
+RUN addgroup -S flood
+RUN adduser -S flood -G flood
+RUN mkdir /data
+RUN chown flood:flood /data
+RUN chown flood:flood ${WORKDIR}
+
+USER flood
+
 COPY --from=nodebuild $WORKDIR $WORKDIR
```

Docker Docs:

> The `USER` instruction sets the user name (or UID) and optionally the user group (or GID) to use when running the image and for any `RUN`, `CMD` and `ENTRYPOINT` instructions that follow it in the Dockerfile.

The `COPY/ADD` instructions creates new files and directories with a UID and GID of 0.

Therefor if we are going to run the app with the `flood` user, I think, we should also copy the files with `flood` owership. There is a optional `--chown` flag to copy using a specified ower, since Docker 17.09.

```suggestion
COPY --from=nodebuild --chown=flood:flood $WORKDIR $WORKDIR
```

---

I will add this suggestion. Then i think that `RUN chown flood:flood /data` can be removed.